packages/web-app-files/tests/unit/helpers/resource/copyMove.spec.ts

```diff
@@ -23,6 +23,22 @@ describe('copyMove', () => {
       webDavPath: '/target'
     }
   })
+
+  it('should name duplicate file correctly', async () => {
```

Comment:
please also add test cases with files / file extensions. probably a good idea to make it a table test (look for `it.each` examples in existing unit tests)